backend/api/views/main.py

```diff
@@ -57,9 +67,12 @@ def get_document():
             ]
         if date is not None:
             docs = [i for i in docs if date.lower() in str(i.date).lower()]
+
+        # if no documents found, let user know
         if len(serialize_list(docs)) == 0:
             return create_response(status=403, message="Query was incorrect")
```

Comment:
I think the message for this can be a little clearer - I feel like it's not necessarily the case that the query was incorrect but simply that no documents were found for that query